commands/commands.go

```diff
@@ -428,6 +441,7 @@ func runActionForeachMachine(actionName string, machines []*libmachine.Host) {
 			serialMachines = append(serialMachines, machine)
 		default:
 			numConcurrentActions++
+			log.Infof("Running action %q on machine %q...", actionName, machine.Name)
```

Comment:
Mmm, personally I like to see the actions which are being taken, in what order they are running, and so on.  Without it, the drivers fend for themselves with messages like "Starting VM...", "Stopping VM...", some of them don't have them, it's inconsistent, etc.